README.md

```diff
@@ -34,6 +42,88 @@ cd snakefmt && poetry install
 poetry shell
 ```
 
+## Example File
+
+Input
+
+```python
+from snakemake.utils import min_version
+min_version("5.14.0")
+configfile: "config.yaml" # snakemake keywords are treated like classes
```

Comment:
I like the additions- here i would add 'treated like classes (2 newlines) maybe, so it's clear why we say this.

The second time you write this (after `include`) could just write` # 2 newlines `

Reply:
:+1: